packages/react-runtime-dev/package.js

```diff
@@ -26,11 +26,13 @@ Npm.depends({
 });
 
 Package.onUse(function (api) {
-  api.use('cosmos:browserify@0.9.2');
+  api.use('ecmascript');
+
+  api.addFiles('client-process.js', 'client');
+  api.export('process', 'client');
```

Comment:
Yes, so that that `process` variable will be declared as a package `var` instead of leaking into the global scope.